lib/test.gi

```diff
@@ -867,7 +867,7 @@ InstallGlobalFunction( "TestDirectory", function(arg)
   totalGcTime := 0;
   
   STOP_TEST_CPY := STOP_TEST;
-  STOP_TEST := function(arg) end;
+  STOP_TEST := function(arg) CallFuncList( STOP_TEST_QUIET, arg ); end;
```

Comment:
```suggestion
  STOP_TEST := STOP_TEST_QUIET;
```

Reply:
This is on purpose: the return value has to be dropped, otherwise it gets printed during the tests.

Reply:
Then perhaps add a comment stating that?